lib/include/prjxray/xilinx/configuration_packetizer.h

```diff
@@ -0,0 +1,72 @@
+#ifndef PRJXRAY_LIB_XILINX_CONFIGURATION_PACKETIZER_H_
+#define PRJXRAY_LIB_XILINX_CONFIGURATION_PACKETIZER_H_
+
+#include <absl/types/optional.h>
+#include <absl/types/span.h>
+#include <prjxray/xilinx/configuration.h>
+
+namespace prjxray {
+namespace xilinx {
+
+template <typename ArchType>
+class ConfigurationPacketizer {
```

Comment:
Class comment

Reply:
To be honest I kept this class because it was there from the beginning, but we don't use it anywhere. I guess I will just remove it with a separate commit so we can restore it easily at any time.